src/riak_kv_ts_svc.erl

```diff
@@ -490,11 +490,11 @@ sub_tsgetreq(Mod, DDL, #tsgetreq{table = Table,
             %% the columns stored in riak_object are just
             %% names; we need names with types, so:
             ColumnTypes = get_column_types(ColumnNames, Mod),
-            {reply, {tsgetresp, {ColumnNames, ColumnTypes, [Row]}}, State};
+            {reply, {tsgetresp, {ColumnNames, ColumnTypes, [list_to_tuple(Row)]}}, State};
```

Comment:
@javajolt - what do you think about moving this conversion to a lower layer?

https://github.com/basho/riak_kv/blob/riak_ts-develop/src/riak_kv_qry_worker.erl#L160-L171